src/common/Throttle.h

```diff
@@ -338,16 +338,58 @@ class TokenBucketThrottle {
   struct Bucket {
     CephContext *cct;
     const std::string name;
-    std::atomic<uint64_t> remain = { 0 }, max = { 0 };
 
-    Bucket(CephContext *cct, const std::string& n, uint64_t m)
-      : cct(cct), name(n),
-	remain(m), max(m)
-    {
-    }
+    // minimum of the filling period.
+    static const uint64_t tick_min = 50;
```

Comment:
As what we discussed offline, make this configurable. hard code as 50ms is not a good idea for all usecase.